tests/L0/run_optimizers/test_dist_adam.py

```diff
@@ -0,0 +1,182 @@
+import argparse
+import random
+import sys
+
+import torch
+from torch.nn.parallel import DistributedDataParallel as DDP
+
+from apex import amp
+from apex.optimizers import FusedAdam
+from apex.contrib.optimizers.distributed_fused_adam_v4 import DistributedFusedAdam
```

Comment:
Could you change this to `distributed_fused_adam` and remove the `_v*` files or are they needed?

Reply:
Done, good catch!